hail/Makefile

```diff
@@ -297,6 +297,7 @@ batch-docs:
 	      BUILDDIR=$(HAIL_DIR)/build/docs/batch \
 	      html
 	rm -rf build/www/docs/batch
+	mkdir -p build/www/docs
```

Comment:
I found this bug. If you run `make -C hail batch-docs` on a fresh repo it will fail because none of these directories exist.